spec/features/saml/ial1_sso_spec.rb

```diff
@@ -91,13 +88,13 @@
       click_submit_default
       click_agree_and_continue
 
-      expect(current_url).to eq saml_authn_request
+      expect(current_url).to eq request_url
     end
   end
 
   context 'fully signed up user authenticates new sp' do
     let(:user) { create(:user, :signed_up) }
-    let(:saml_authn_request) { auth_request.create(saml_settings) }
+    let(:saml_authn_request) { saml_authn_request_url }
```

Comment:
is this a request object or a string url, also looks like all the specs were changed to use `saml_authn_request_url` directly so can we remove it?

Reply:
It's just a string url, thus the name change on the helper method (was `authn_request`). This particular context is still using `saml_authn_request` throughout - maybe I'm not seeing what you are?